### ISSUE TITLE
JSDelivr @latest is pulling the 7.0.0-rc version

### ISSUE THREAD

**Issue by dxblouie** [fiduswriter/simple-datatables#246]
While it's a bad practice to use @latest tag on a production server as you've already noted in the documentation, i did miss one, and spent an hour debugging a site of mine this morning when the tables began showing urlencoded cell content (for instance a cell with an a href tag, was literally showing "<a href=....>" as in "&lt;a href=.....&gt;")

in any case, it turned out to be jsdelivr @latest is pulling the rc version, which i find is odd.
nonetheless, i've switched back to @6 for now.. and i thought i'd share here in case it may help someone.

**Comment by supyrow** [comment]
yeah...  i was losing my mind.

**Comment by johanneswilm** [comment]
Yeah, sorry about that. I tried to add a large notice on the main page about it when version 6 came. Anyway, you'll have to do something about the tag version 7 final is released, which will likely be quite soon.

Version 7.0.0-rc.1 (just released) behaves a little more similar to version 6 in terms of showing HTML by default,. So hopefully that also means fewer problems.

I will keep this issue open until version 7 has been released.
